Automattic/wp-calypso
# Importer: Update Medium icon

#### Changes proposed in this Pull Request

* Replaces the icon for the Medium importer with up-to-date Medium branding.

Before:

<img width="726" alt="image" src="https://user-images.githubusercontent.com/1647564/56662369-b5f1d680-669b-11e9-89ef-ca39b3899dec.png">

After: 

<img width="727" alt="image" src="https://user-images.githubusercontent.com/1647564/56662389-c1450200-669b-11e9-9e20-ee50e4d27ab1.png">

#### Testing instructions

* Check the branch out and start local Calypso.
* Go to the imports page http://calypso.localhost:3000/settings/import/[ site ].
* The Medium importer should have the new branding.
* Click on the "Start Import" button in the Medium importer.
* The expanded importer should have the new branding.

Fixes #
https://github.com/Automattic/wp-calypso/issues/27816
